pkg/rpc/client/rpc.go

```diff
@@ -113,6 +241,30 @@ func (c *Client) GetNEP5Transfers(address string) (*result.NEP5Transfers, error)
 	return resp, nil
 }
 
+// GetPeers returns the list of nodes that the node is currently connected/disconnected from.
+func (c *Client) GetPeers() (*result.GetPeers, error) {
+	var (
+		params = request.NewRawParams()
+		resp   = &result.GetPeers{}
+	)
+	if err := c.performRequest("getpeers", params, resp); err != nil {
+		return resp, err
+	}
+	return resp, nil
+}
+
+// GetRawMemPool returns the list of unconfirmed transactions in memory.
+func (c *Client) GetRawMemPool() (*[]util.Uint256, error) {
```

Comment:
I think it should return `([]util.Uint256, error)`, slice is a reference type already, it doesn't make much sense returning a pointer to it here.